packages/syntax-floating-components/src/index.tsx

```diff
@@ -0,0 +1,3 @@
+import { Tooltip, TooltipContent, TooltipTrigger } from "./Tooltip/Tooltip";
+
+export { Tooltip, TooltipContent, TooltipTrigger };
```

Comment:
@JFChoe Can we make sure the tooltip opens when you focus on a button inside of it?

Reply:
I added an uncontrolled button. When you focus on it, the tooltip opens up.